experimental/framework/tinylicious-client/src/test/TinyliciousClient.spec.ts

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import { v4 as uuid } from "uuid";
+import { SharedMap } from "@fluid-experimental/fluid-framework";
+import { ContainerSchema } from "@fluid-experimental/fluid-static";
+import {
+    TinyliciousClient,
+    TinyliciousConnectionConfig,
+    TinyliciousContainerConfig,
+} from "..";
+
+describe("TinyliciousClient", () => {
+    before(() => {
+        const clientConfig: TinyliciousConnectionConfig = { port: 7070 };
+        TinyliciousClient.init(clientConfig);
```

Comment:
Does this statement launch tinylicious process?

Reply:
If so, should it be terminated in a test cleanup routine?

Reply:
Tinylicious is launched in the npm command in the package json (the `start-server-and-test` command, which also handles terminating it).  This call sets up the test's connection to the tinylicious process.